moby/moby
# test: Remove DOCKER_ENGINE_GOARCH from the tests

**- What I did**

Instead of looking at the `DOCKER_ENGINE_GOARCH` env var that's not even plumbed down to the tests we use the daemon version info from `testEnv`

**- How I did it**

...

**- How to verify it**

Run the `TestPluginLogDriver` test on an AMD64 machine, it should now run and not be skipped.

On the CI,

Before:

```
=== RUN   TestDockerCLIPluginLogDriverSuite
=== RUN   TestDockerCLIPluginLogDriverSuite/TestPluginLogDriver
    docker_cli_plugins_logdriver_test.go:25: unmatched requirement IsAmd64
=== RUN   TestDockerCLIPluginLogDriverSuite/TestPluginLogDriverInfoList
    docker_cli_plugins_logdriver_test.go:45: unmatched requirement IsAmd64
--- PASS: TestDockerCLIPluginLogDriverSuite (0.01s)
    --- SKIP: TestDockerCLIPluginLogDriverSuite/TestPluginLogDriver (0.00s)
    --- SKIP: TestDockerCLIPluginLogDriverSuite/TestPluginLogDriverInfoList (0.00s)
PASS
```

After:

```
=== RUN   TestDockerCLIPluginLogDriverSuite
=== RUN   TestDockerCLIPluginLogDriverSuite/TestPluginLogDriver
=== RUN   TestDockerCLIPluginLogDriverSuite/TestPluginLogDriverInfoList
--- PASS: TestDockerCLIPluginLogDriverSuite (10.64s)
    --- PASS: TestDockerCLIPluginLogDriverSuite/TestPluginLogDriver (5.65s)
    --- PASS: TestDockerCLIPluginLogDriverSuite/TestPluginLogDriverInfoList (4.99s)
PASS
```

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
